tests/test-cases.js

```diff
@@ -1691,6 +1709,18 @@ var PARSE_ASYNC_TESTS = [
 			errors: []
 		}
 	},
+	{
+		description: "Simple file with BOM encoding and header",
```

Comment:
I do not think this test is necessary as its already tested by previous ones.
So I think we can just remove it.